semantic_conventions/resource/telemetry.yaml

```
@@ -45,3 +45,13 @@ groups:
         brief: >
           The version string of the auto instrumentation agent, if used.
         examples: ["1.2.3"]
+      - id: distro.name
+        type: string
+        brief: >
+          The name string of the distribution, if used.
```

The "if used" is unclear.

We need to define here what constitutes a "distribution" and what not. E.g. is opentelemetry-java, as uploaded to Maven Central, with all the BOM files, etc., a distribution? Or is only the agent a distribution?

It seems you added a glossary entry, which is good. Please link to it.

And nitpicking some more: "used" is still the wrong word here, probably should use

```suggestion
          The name of the distribution, if applicable.
```

---

Unresolving, there is more to this comment than just this suggestion (and the suggestion also applies to distro.version)